pyatlan/model/packages/asset_export(basic).py

```diff
@@ -0,0 +1,290 @@
+            {
+                "name": f"csa-{self._NAME}-{self._epoch}-0",
+                "auth_type": "gcs",
+                "username": project_id,
+                "password": service_account_json,
+                "extra": {
+                    "gcs_bucket": bucket,
+                },
+                "connector_config_name": "csa-connectors-objectstore",
+            }
+        )
+        return self
+
+    def adls(
+        self,
+        client_id: str,
+        client_secret: str,
+        tenant_id: str,
+        account_name: str,
+        container: str,
+    ) -> AssetExportPackage:
+        """
+        Set up package to export to Azure Data Lake Storage.
+        :param client_id: Azure AD client ID
+        :param client_secret: Azure AD client secret
+        :param tenant_id: Azure tenant ID
+        :param account_name: ADLS storage account name
+        :param container: ADLS container to upload the export file to
+        """
+        self._credentials_body.update(
+            {
+                "name": f"csa-{self._NAME}-{self._epoch}-0",
+                "auth_type": "adls",
+                "username": client_id,
+                "password": client_secret,
+                "extra": {
+                    "azure_tenant_id": tenant_id,
+                    "storage_account_name": account_name,
+                    "adls_container": container,
+                },
+                "connector_config_name": "csa-connectors-objectstore",
+            }
+        )
+        return self
+
+    def _add_delivery_parameters(self):
+        """
+        Add delivery parameters to the parameters list.
+        """
+        self._parameters.append(
+            {
+                "name": "delivery_type",
+                "value": self._delivery_type,
+            }
+        )
+        if self._delivery_type == "EMAIL" and self._email_addresses:
+            self._parameters.append(
+                {
+                    "name": "email_addresses",
+                    "value": ",".join(
+                        self._email_addresses
+                    ),  # Join the email addresses if they are in a list
+                }
+            )
+
+    def _get_metadata(self) -> WorkflowMetadata:
+        return WorkflowMetadata(
+            labels={
+                "orchestration.atlan.com/certified": "true",
+                "orchestration.atlan.com/preview": "true",
+                "orchestration.atlan.com/source": self._NAME,
+                "orchestration.atlan.com/sourceCategory": "utility",
+                "orchestration.atlan.com/type": "custom",
+                "orchestration.atlan.com/verified": "true",
+                "package.argoproj.io/installer": "argopm",
+                "package.argoproj.io/name": "a-t-rcsas-l-a-s-h{self._NAME}",
+                "package.argoproj.io/parent": "",
+                "package.argoproj.io/registry": "local",
+                "package.argoproj.io/version": "2.3.1-SNAPSHOT",
```

Comment:
Let's remove `parent` and `version` and define only `registry` as shown below (similar to [other packages](https://github.com/atlanhq/atlan-python/blob/00f5a3be100951f4c7fbe6c1675fe6c43ff88b1f/pyatlan/model/packages/asset_import.py#L362)). This way, whenever someone creates package workflows through the SDK, it will pick the latest version instead of explicitly mentioning a version, ex: `2.3.1-SNAPSHOT`. (Please ensure you test manually 🧪  after this change to confirm that it picks the latest package version ✅):

```json
"package.argoproj.io/registry": "httpsc-o-l-o-ns-l-a-s-hs-l-a-s-hpackages.atlan.com",
```